dandi/girder.py

```diff
@@ -556,6 +556,71 @@ def lock_dandiset(self, dandiset_identifier: str):
                         f"Failed to unlock dandiset {dandiset_identifier} due to: {msg}"
                     )
 
+    NGINX_MAX_CHUNK_SIZE = 400 * (1 << 20)  # 100 MiB
```

Comment:
Yarik fell into its own trap ! ;)
```suggestion
    NGINX_MAX_CHUNK_SIZE = 400 * (1 << 20)  # 400 MiB
```

Reply:
I will merge locally and force push